imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -147,17 +146,18 @@ class CaseExplorer extends Component {
     }
   )
   render () {
-    const { isLoading, caseList, allNotifications, unreadNotifications } = this.props
+    const { isLoading, caseList, allNotifications, unreadNotifications, searchResult } = this.props
     const { selectedRoleFilter, sortBy, open } = this.state
     if (isLoading) return <Preloader />
-
-    const caseGrouping = this.makeCaseGrouping({
-      caseList,
-      selectedRoleFilter,
-      sortBy,
-      allNotifs: allNotifications,
-      unreadNotifs: unreadNotifications
-    })
+    const cases = this.props.searchText === '' ? caseList : searchResult
+    const caseGrouping =
+        this.makeCaseGrouping({
+          cases,
```

Comment:
searchResult returns undefined in makeCaseGrouping function.

Reply:
Sorry, I still don't understand what this means...